resources/assets/enderio/lang/zh_CN.lang

```diff
@@ -8,30 +8,23 @@ enderio.power.maxOutput=最大输出
 // pre-formatted number, "RF" ("power.rf"), pre-formatted number, "RF" ("power.rf"), "/t" ("power.tick")
 enderio.power.generates=产能：{0}{1}，产能速率{2}{3}{4}
 
-//Ender IO / Enderface
-tile.blockEnderIo.name=末影接口
-tile.blockEnderIo.tooltip.detailed.line1=允许远程访问附近的
-tile.blockEnderIo.tooltip.detailed.line2=方块
-tile.blockEnderIo.tooltip.detailed.line3=使用旅行锚或
-tile.blockEnderIo.tooltip.detailed.line4=旅行权杖
-
+//Enderface
 item.enderio.itemEnderface.name=末影面板
 
 //Conduits
 enderio.blockConduitBundle=导管接口
-enderio.blockConduitFacade=导管框架
-item.enderio.itemConduitFacade.name=导管框架
-item.enderio.itemConduitFacade.hardened.name=硬化导管框架
-item.enderio.itemConduitFacade.transparent.name=透明导管框架
-item.enderio.itemConduitFacade.transparent.hardened.name=硬化透明导管框架
+item.itemConduitFacade.name=导管框架
+item.itemConduitFacade.hardened.name=硬化导管框架
+item.itemConduitFacade.transparent.name=透明导管框架
+item.itemConduitFacade.transparent.hardened.name=硬化透明导管框架
 enderio.item.itemConduitFacade.tooltip.notpainted=未喷涂
-item.enderio.itemConduitFacade.tooltip.detailed.line1=可放入喷涂机喷涂
-item.enderio.itemConduitFacade.tooltip.detailed.line2=右击一个导管可放置
-item.enderio.itemConduitFacade.tooltip.detailed.line3=当手里拿着一个扳手或者导管时 
-item.enderio.itemConduitFacade.tooltip.detailed.line4=导管框架隐藏
-item.enderio.itemConduitFacade.hardened.tooltip=硬化: 抵抗破坏以及爆炸
-item.enderio.itemConduitFacade.transparent.tooltip=透明:当使用非固体方块喷涂时，隐藏导管
-item.enderio.itemConduitFacade.transparent.hardened.tooltip=透明:当使用非固体方块喷涂时，隐藏导管；硬化: 抵抗破坏以及爆炸
+item.itemConduitFacade.tooltip.detailed.line1=可放入喷涂机喷涂
+item.itemConduitFacade.tooltip.detailed.line2=右击一个导管可放置
+item.itemConduitFacade.tooltip.detailed.line3=当手里拿着一个扳手或者导管时 
+item.itemConduitFacade.tooltip.detailed.line4=导管框架隐藏
+item.itemConduitFacade.hardened.tooltip=Hardened=硬化: 抵抗破坏以及爆炸
+item.itemConduitFacade.transparent.tooltip=Transparent=透明:当使用非固体方块喷涂时，隐藏导管
+item.itemConduitFacade.transparent.hardened.tooltip=Transparent=透明:当使用非固体方块喷涂时，隐藏导管；硬化: 抵抗破坏以及爆炸
```

Comment:
These "Hardened="/"Transparent=" are part of the text, I don't think you want them there.